addons/l10n_ro_efactura/data/neutralize.sql

```diff
@@ -0,0 +1,5 @@
+UPDATE res_company
+    SET l10n_ro_edi_test_env = true,
+        l10n_ro_edi_client_secret = 'dummy_secret',
+        l10n_ro_edi_access_token = 'dummy_token';
```

Comment:
We should also update client_id and refresh_token too. And I think false (null) value is better.
```suggestion
        l10n_ro_edi_client_id = false,
        l10n_ro_edi_client_secret = false,
        l10n_ro_edi_access_token = false,
        l10n_ro_edi_refresh_token = false;
```
If we have to make a request, we just need to fill `l10n_ro_edi_access_token`, we can leave the other empty

Reply:
ok, done.

I went for a real NULL though rather than a false. 
Let's avoid putting Booleans into varchar columns ^^